src/lib/components/common/Tip.svelte

```diff
@@ -56,7 +57,8 @@
     border-color: var(--green-2);
     background: var(--green-1);
   }
-  .tip.danger {
+  .tip.danger,
+  .tip.error {
```

Comment:
Why add the `error` mode if it's using the same styling as `danger`?

If we want to keep it, could set `error` to use `--red-n` variables in place of orange.